guides/common/modules/proc_modifying-a-smart-class-parameter-by-using-an-external-file.adoc

```diff
@@ -0,0 +1,140 @@
+[id="modifying-a-smart-class-parameter-by-using-an-external-file"]
+= Modifying a Smart Class parameter by using an external file
+
+Using external files simplifies working with JSON data.
```

Comment:
Introduction does not really relate to the heading; although I don't really have a good suggestion for you. Not blocking this PR.